Uchu.World/Systems/Missions/MissionTaskInstance.cs

```diff
@@ -204,7 +204,7 @@ private void MessageUpdateMissionTask()
             {
                 Associate = Mission.Player,
                 MissionId = Mission.MissionId,
-                TaskIndex = MissionTaskIndex,
+                TaskIndex = 1 << (MissionTaskIndex + 1),
```

Comment:
[`MissionTaskInstance`](https://github.com/UchuServer/Uchu/blob/c064bd2240a5a16c3b0ff5a8c5803a826d72c968/Uchu.World/Packets/GameMessages/Server/NotifyMissionTaskMessage.cs) currently handles this bit manipulation internally, but that functionality is removed. An enhancement in the future may require a finalizer function, but this is the only case for now. It could be implemented in this pull request in a future one though.

Reply:
I see, thanks for explaining